install_helper.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# install_helper allows for easy installation of code-server on Linux systems
+# the latest official release is pulled from the Github API, where the proper asset
+# is then unpackaged and installed into ~/.local/share/code-server/<version>
+
+# The code-server binary is linked to ~/.local/share/code-server/bin/code-server
+
+set -euo pipefail
+
+RED=1
```

Comment:
I would put this into a function instead of a variable just so you don't have to remember `tput setaf` everywhere else.